Yosemite/Yosemite/Base/SessionManagerProtocol.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import Observables
+
+public protocol SessionManagerProtocol {
+    var defaultAccount: Account? { get set}
+    var defaultAccountID: Int64? { get }
+
+    var defaultSite: Site? { get set }
+    var defaultStoreID: Int64? { get set }
+
+    var anonymousUserID: String? { get }
+
+    var siteID: Observable<Int64?> { get }
+
+    var defaultCredentials: Credentials? { get set}
+
+    func reset()
+}
```

Comment:
⚠️ Would you mind copying over the comments from the `SessionManager`? 🙂 Since this is referred to by `StoresManager`, I think it makes more sense to have some comments here.

Reply:
For sure – happy to do so! The reason I didn't previously was it seemed like most `Yosemite` protocols didn't have this, and I assumed it was to avoid maintaining both 😅 

I think I like it better this way 🙂

Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3305/commits/b4f525d7d6b65d0aa56856747603d1cc85c6514f